diagnostic.md

```diff
@@ -36,13 +42,21 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com/donuts/$ID
 ```
 
 Write an AJAX request to retrieve a **single donut** from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonut =
+
+$.ajax({
+   type: "GET",
+   url: "http://www.example.com" + "/donuts";
```

Comment:
The ID here is actually attached to the end of the url, so something like this: `http://www.example.com" + "/donuts/4`